ACE/ace/Ping_Socket.cpp

```
@@ -222,11 +224,12 @@ ACE_Ping_Socket::process_incoming_dgram (char * ptr, ssize_t len)
 
   if ((icmplen = len - hlen1) < ICMP_MIN)
     {
-      ACELIB_DEBUG
-        ((LM_DEBUG,
-          ACE_TEXT ("(%P|%t) ACE_Ping_Socket::process_incoming_dgram")
-          ACE_TEXT (" - ICMP length is %b < 8.\n"),
-          icmplen));
+      if (ACE::debug())
+        ACELIB_DEBUG
+          ((LM_DEBUG,
+            ACE_TEXT ("(%P|%t) ACE_Ping_Socket::process_incoming_dgram")
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If ACE_TEXT is a macro then please configure it.

(unknownMacro)